pettingzoo/mpe/scenarios/group_spread.py

```diff
@@ -1,91 +1,155 @@
+from enum import Enum
+
 import numpy as np
+
 from .._mpe_utils.core import World, Agent, Landmark
 from .._mpe_utils.scenario import BaseScenario
-import random
+
+
+# Predefined colors for agents and landmarks
+COLORS = [
+    np.array([0.25, 0.25, 0.25]),
+    np.array([0.85, 0.25, 0.25]),
+    np.array([0.25, 0.85, 0.25]),
+    np.array([0.25, 0.25, 0.85]),
+    np.array([0.85, 0.85, 0.25]),
+    np.array([0.85, 0.25, 0.85]),
+    np.array([0.25, 0.85, 0.85]),
+    np.array([1.00, 0.00, 0.00]),
+    np.array([0.00, 1.00, 0.00]),
+    np.array([0.00, 0.00, 1.00]),
+]
```

Comment:
To avoid having all grey-scale colors if fixed set of colors are used, I hand-wrote this list of possible colors. Main downside is we are now limited to max 10 colors but it's easy to extend if there would be a need.

Reply:
Looks good to me for now, unlikely that we will have more than 10 groups but I'll keep it in mind if there's a way for a more automated approach